ably/metadata.csv

```diff
@@ -0,0 +1,155 @@
+ably.apiRequests.push.failed,count,60,request,,Total number of failed Push requests.,0,ably,apiRequests.push.failed,
+ably.apiRequests.push.refused,count,60,request,,Total number of refused Push requests.,0,ably,apiRequests.push.refused,
+ably.apiRequests.push.succeeded,count,60,request,,Total number of Push requests made.,0,ably,apiRequests.push.succeeded,
+ably.apiRequests.tokenRequests.failed,count,60,request,,Total number of failed token requests.,0,ably,apiRequests.tokenRequests.failed,
+ably.apiRequests.tokenRequests.refused,count,60,request,,Total number of refused token requests (due to permissions or rate limiting).,0,ably,apiRequests.tokenRequests.refused,
+ably.apiRequests.tokenRequests.succeeded,count,60,request,,Total number of token requests made.,0,ably,apiRequests.tokenRequests.succeeded,
+ably.channels.mean,gauge,60,item,,Mean active channel count.,0,ably,channels.mean,
+ably.channels.min,gauge,60,item,,Minimum active channel count.,0,ably,channels.min,
+ably.channels.opened,count,60,item,,Total number of channels opened.,0,ably,channels.opened,
+ably.channels.peak,gauge,60,item,,Peak active channel count.,0,ably,channels.peak,
+ably.channels.refused,count,60,request,,Total number of channel attach requests that failed because of permissions.,0,ably,channels.refused,
+ably.connections.all.mean,gauge,60,connection,,Mean connection count.,0,ably,connections.all.mean,
+ably.connections.all.min,gauge,60,connection,,Minimum connection count.,0,ably,connections.all.min,
+ably.connections.all.opened,count,60,connection,,Total number of connections opened.,0,ably,connections.all.opened,
+ably.connections.all.peak,gauge,60,connection,,Peak connection count.,0,ably,connections.all.peak,
+ably.connections.all.refused,count,60,connection,,Total number of connections refused.,0,ably,connections.all.refused,
+ably.messages.all.all.count,count,60,message,,Total message count.,0,ably,messages.all.all.count,
+ably.messages.all.all.data,count,60,bytes,,Total message size.,0,ably,messages.all.all.data,
+ably.messages.all.all.uncompressedData,count,60,bytes,,"Total uncompressed message size (excluding any compression, e.g. delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas).",0,ably,messages.all.all.uncompressedData,
+ably.messages.all.messages.count,count,60,message,,"Total message count, excluding presence messages.",0,ably,messages.all.messages.count,
+ably.messages.all.messages.data,count,60,bytes,,"Total message size, excluding presence messages.",0,ably,messages.all.messages.data,
+ably.messages.all.messages.uncompressedData,count,60,bytes,,"Total uncompressed message size (excluding any compression, e.g. delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas), excluding presence messages.",0,ably,messages.all.messages.uncompressedData,
```

Comment:
```suggestion
ably.messages.all.all.uncompressedData,count,60,bytes,,Total uncompressed message size (excluding any compression, such as delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas).,0,ably,messages.all.all.uncompressedData,
ably.messages.all.messages.count,count,60,message,,Total message count, excluding presence messages.,0,ably,messages.all.messages.count,
ably.messages.all.messages.data,count,60,bytes,,Total message size, excluding presence messages.,0,ably,messages.all.messages.data,
ably.messages.all.messages.uncompressedData,count,60,bytes,,Total uncompressed message size (excluding any compression, such as delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas), excluding presence messages.,0,ably,messages.all.messages.uncompressedData,
```

Reply:
Updated in b2d3976664216d195c2ad1adf80bef8f5f3447b2